zhazam/Modules/Loading/LoadingViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LoadingViewController.swift
+//  zhazam
+//
+//  Created by Nurbek Ismagulov on 8/20/19.
+//  Copyright © 2019 Nurbek Ismagulov. All rights reserved.
+//
+
+import UIKit
+
+final class LoadingViewController: UIViewController {
+    enum LocalConstants {
+        static let loadedPercentage: Int = 10
```

Comment:
Correct me if I am wrong - is this constant is used to indicate the upper bound for possible percentage value?
Maybe I am over-doing it here: but tbh it causes an impression that this one is used to show loaded **fraction** of maximum possible percentage. 

Consider the case "Loading... 33%":
- Here we can say that "33%" is **loadedPercentage** - that's what I am talking about